bundles/org.openhab.transform.scale/conf/transform/scale/catchnonnumeric.scale

```diff
@@ -0,0 +1,5 @@
+[-8..10[=low
+[10..20[=middle
+[20..300[=high
+[300..]=extreme
+non-number=Non Numeric
```

Comment:
I like the idea but how can this happen?

Reply:
Yes, this happens when values changes to NULL or UNDEF, to log file is plenty of warnings stating that Scale can only be applied to numeric values. In this case, if the user decides it, no warning and a nice output.

Reply:
I'm not so sure about the naming, though. I would prefer `invalid`, `NaN`, `undefined` or something like that. Or: can we re-use `[..]` for undefined ranges for that?